llvm/test/CodeGen/AMDGPU/flat-scratch-fold-fi.mir

```diff
@@ -1,5 +1,6 @@
 # NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
 # RUN: llc -mtriple=amdgcn -mcpu=gfx900 -mattr=+enable-flat-scratch -run-pass=si-fold-operands -verify-machineinstrs -o - %s | FileCheck -check-prefix=GCN %s
+# RUN: llc -mtriple=amdgcn -mcpu=gfx900 -mattr=+enable-flat-scratch -passes=si-fold-operands -verify-machineinstrs -o - %s | FileCheck -check-prefix=GCN %s
```

Comment:
I don't want to double the number of run lines to test the pass itself. Just have a couple with the two forms? If you want to update all the tests, you can just use the new form

Reply:
Alright, makes sense.